be/src/exec/tablet_sink_index_channel.cpp

```diff
@@ -758,6 +760,19 @@ Status NodeChannel::_wait_request(ReusableClosure<PTabletWriterAddBatchResult>*
                   << " commit " << _tablet_commit_infos.size() << " tablets: " << commit_tablet_id_list_str;
     }
 
+    if (closure->result.has_load_channel_profile()) {
```

Comment:
it seems that no need to wrap a thrift object into protobuf object, can we just use either thrift or protobuf instead of both.

Reply:
LoadChannel and OlapTableSinkOperator use protobuf for communication, but there is not a protobuf structure for `RuntimeProfile` like `RuntimeProfile.thrift`. We can add a structure for protobuf, but there is burden to maintain these protobuf/thrift structures. So currently just wrap a thrift object into protobuf object.